kfs-core/src/main/java/edu/arizona/kfs/tax/businessobject/Payee.java

```diff
@@ -0,0 +1,246 @@
+	private Integer vendorHeaderGeneratedIdentifier; // VNDR_HDR_GNRTD_ID
+	private Integer vendorDetailAssignedIdentifier; // VNDR_DTL_ASND_ID
+
+	private VendorDetail vendorDetail;
+
+	public Payee() {
+
+	}
+
+	public Integer getId() {
+		return id;
+	}
+
+	public void setId(Integer id) {
+		this.id = id;
+	}
+
+	public String getHeaderTypeCode() {
+		return headerTypeCode;
+	}
+
+	public void setHeaderTypeCode(String headerTypeCode) {
+		this.headerTypeCode = headerTypeCode;
+	}
+
+	public String getHeaderTaxNumber() {
+		return headerTaxNumber;
+	}
+
+	public void setHeaderTaxNumber(String headerTaxNumber) {
+		this.headerTaxNumber = headerTaxNumber;
+	}
+
+	public String getHeaderOwnershipCode() {
+		return headerOwnershipCode;
+	}
+
+	public void setHeaderOwnershipCode(String headerOwnershipCode) {
+		this.headerOwnershipCode = headerOwnershipCode;
+	}
+
+	public String getHeaderOwnershipCategoryCode() {
+		return headerOwnershipCategoryCode;
+	}
+
+	public void setHeaderOwnershipCategoryCode(String headerOwnershipCategoryCode) {
+		this.headerOwnershipCategoryCode = headerOwnershipCategoryCode;
+	}
+
+	public String getVendorName() {
+		return vendorName;
+	}
+
+	public void setVendorName(String vendorName) {
+		this.vendorName = vendorName;
+	}
+
+	public String getVendorNumber() {
+		return vendorNumber;
+	}
+
+	public void setVendorNumber(String vendorNumber) {
+		this.vendorNumber = vendorNumber;
+	}
+
+	public String getAddressTypeCode() {
+		return addressTypeCode;
+	}
+
+	public void setAddressTypeCode(String addressTypeCode) {
+		this.addressTypeCode = addressTypeCode;
+	}
+
+	public String getAddressLine1Address() {
+		return addressLine1Address;
+	}
+
+	public void setAddressLine1Address(String addressLine1Address) {
+		this.addressLine1Address = addressLine1Address;
+	}
+
+	public String getAddressLine2Address() {
+		return addressLine2Address;
+	}
+
+	public void setAddressLine2Address(String addressLine2Address) {
+		this.addressLine2Address = addressLine2Address;
+	}
+
+	public String getAddressCityName() {
+		return addressCityName;
+	}
+
+	public void setAddressCityName(String addressCityName) {
+		this.addressCityName = addressCityName;
+	}
+
+	public String getAddressStateCode() {
+		return addressStateCode;
+	}
+
+	public void setAddressStateCode(String addressStateCode) {
+		this.addressStateCode = addressStateCode;
+	}
+
+	public String getAddressZipCode() {
+		return addressZipCode;
+	}
+
+	public void setAddressZipCode(String addressZipCode) {
+		this.addressZipCode = addressZipCode;
+	}
+
+	public String getAddressCountryCode() {
+		return addressCountryCode;
+	}
+
+	public void setAddressCountryCode(String addressCountryCode) {
+		this.addressCountryCode = addressCountryCode;
+	}
+
+	public Integer getTaxYear() {
+		return taxYear;
+	}
+
+	public void setTaxYear(Integer taxYear) {
+		this.taxYear = taxYear;
+	}
+
+	public Boolean getVendorForeignIndicator() {
+		return vendorForeignIndicator;
```

Comment:
Usually would want getters for Boolean values to be of the form .isSomeState, however the field names here don't make it obvious that the backing type is a Boolean... for instance a "vendor foreign indicator" could just as easily be a String or an int as a boolean. Since the field names here are generic, sticking with 'getFieldName' makes more sense to me than shoehorning in the grammar of 'isSomeState'.